### PR TITLE
feat: example modal and teleport

### PR DESCRIPTION
**Pull request by BrocksiNet**
### Description

closes: #64

### Type of change

- [X] Example Implementation

### Additional context

Look at the Readme in this PR.

### CODE REVIEW

File: examples/modal-teleport/components/AppModal.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            leave-to-class="opacity-0 translate-y-10 translate-y-0 scale-95"
+          >
+            <div
+              class="bg-white rounded-lg text-left overflow-hidden shadow-xl p-8 w-1/2"
+              role="dialog"
+              ref="modal"
+              aria-modal="true"
+              aria-labelledby="modal-headline"
+              id="modal-content"
+              @click.stop
```
Comment (patzick) [inline examples/modal-teleport/components/AppModal.vue:42]:
```suggestion
              v-if="isOpen"
```

Comment (patzick) [reply examples/modal-teleport/components/AppModal.vue:42]: to avoid mounting slot before modal is opened

**Review by patzick — APPROVED** [review]

**Comment by BrocksiNet** [comment]
I guess we can close this PR because https://github.com/shopware/frontends/pull/165 is already merged.

**Comment by patzick** [comment]
@bojanrajh #165 was merged into this PR ;D

**Pull request merged**